index.js

```diff
@@ -59,6 +64,7 @@ class MarkdownHeaderButtonElement extends MarkdownButtonElement {
 if (!window.customElements.get('md-header')) {
   window.MarkdownHeaderButtonElement = MarkdownHeaderButtonElement
   window.customElements.define('md-header', MarkdownHeaderButtonElement)
+  buttonSelectors.push('md-header')
```

Comment:
Can these selectors be added to the array's initializer so the array isn't changed after definition? These elements aren't conditionally added to registry.

Reply:
They can but I figured this was more maintainable, as it they're added in place which  means should a new element be added, it won't be so easy to forget to add it to the selector map.